source/uwp/SharedVisualizer/CustomElements/CustomInput.cs

```diff
@@ -5,7 +5,14 @@
 using System.Linq;
 using Windows.Data.Json;
 
-#if !USE_WINUI3
+#if USE_WINUI3
+using AdaptiveCards.Rendering.Winui3;
```

Comment:
nit, can we please fix the capitalization :) so WinUI3 instead of Winui3

Reply:
https://github.com/microsoft/AdaptiveCards/pull/8255